malpolon/models/standard_prediction_systems.py

```diff
@@ -448,4 +456,92 @@ def __init__(
                              'kwargs': {}}
             }
 
+        super().__init__(model, loss, optimizer, metrics)
+
+
+class RegressionSystem(GenericPredictionSystem):
+    """Regression task class."""
+    def __init__(
```

Comment:
Some arguments  (optimizer kwargs) are not used because incompatible with the proposed default optimizer. I suggest replacing them with the default optimizer's; or simply removing them.

Please update the docstring accordingly too

Reply:
removed the kwargs